src/tests/system/tests/test_sssctl.py

```diff
@@ -764,39 +508,27 @@ def test_sssctl__analyze_child_logs(client: Client, ipa: IPA):
 
 
 @pytest.mark.importance("medium")
-@pytest.mark.tools
 @pytest.mark.ticket(bz=[2142960, 2142794, 2142961])
+@pytest.mark.tools
 @pytest.mark.topology(KnownTopology.LDAP)
-def test_sssctl__analyze_without_root_privileges(client: Client, ldap: LDAP):
+def test_sssctl__analyze_without_root_privileges(client: Client, provider: GenericProvider):
```

Comment:
What is this actually testing?
There are IMHO some more or less questionabe use-cases:
1. non-privileged user is able to analyze logs where they are (via suid)  - Probably unwanted behaviour as it might leak confidental information.
2. root user copies logs somewhere and change ownership to non-root user1. What is the point to do that instead of running analyze directly as root?
3. Logs can be analyzed in **any given location** by **anyone who can read** them.

I think we are covering 3. there so I would ephasize that we are also covering "--logdir" in the testcase.